tasks/tls_signed.yml

```diff
@@ -7,18 +7,44 @@
   ansible.builtin.set_fact:
     nextcloud_tls_cert_key_file: "{{ nextcloud_tls_cert_key | default(\"/etc/ssl/\" + nextcloud_instance_name + \".key\") }}"
 
-- name: "[SIGNED TLS] - Certificate is on the host"
+- name: "[SIGNED TLS] - Define certificate chain path"
+  ansible.builtin.set_fact:
+    nextcloud_tls_chain_file: "{{ nextcloud_tls_chain_file | default(\"/etc/ssl/\" + nextcloud_instance_name + \".pem\") }}"
+  when: nextcloud_tls_src_chain is defined
```

Comment:
True. These are _new_ variables, which are necessary with this change. And If I add the same feature for certificate-generation, then this will be equally required. Root-CA Certificate => Intermediate CA => Signed Server Certfiicate is best practice.

Reply:
This pertains to update README, which I'll do once the Pull Request is accepted.

Reply:
Please update the defaults and the readme in the PR. :smiley:

Reply:
Done.

Reply:
`nextcloud_tls_chain_file` is still undefined. Please add it to defaults and README.

Reply:
Changed.